lib/gateway/actionParams.js

```diff
@@ -0,0 +1,30 @@
+const log = require('../logger').policy;
+class ActionParams {
```

Comment:
And here you are saying to create a class https://github.com/ExpressGateway/express-gateway/pull/544#discussion_r151945456
which one is your final approach and why it is better ?

Reply:
Ok — but I would still try to find a more precise name. `commonAuth` sounds to me like "Hey let's put everything you need here". I'm worried it could become a mega-monster-giant method with the passing of the time. Scoped named methods discourage this bad practice, as your code might not fit with what the function is claiming to do.

Again, it's a nitpick, we can definitely proceed without touching this code at all.